src/core/gosling-embed.ts

```diff
@@ -20,6 +20,7 @@ export type GoslingEmbedOptions = Omit<HiGlassComponentWrapperProps['options'],
     id?: string;
     className?: string;
     theme?: Theme;
+    urlToFetchOptions?: UrlToFetchOptions;
```

Comment:
I assume you are also planning to update `GoslingCompProps` of `GoslingComponent` by adding `urlToFetchOptions`.

Reply:
Oops I forgot about that! Yes just added now.